### ISSUE TITLE
Cron script failing on AWS

### ISSUE THREAD

**Issue by mcapuccini** [kubenow/KubeNow#379]
https://travis-ci.org/kubenow/cron/builds/370664014

**Comment by carmat88** [comment]
Indeed, it's again that annoying issue related to the latest version of PIP. Will roll back to 9.0.3 here as well and actually install openstack-cli instead of glance so to use more updated tools.
